### ISSUE TITLE
Wrong app-port on PubSub .NET Quickstart

### ISSUE THREAD

**Issue by k3daevin** [dapr/docs#3579]
**Describe the issue**
The command to run the order-processor says `--app-port 7005`
Since the app runs on 7006, the subscription does not work.

**URL of the docs**
https://docs.dapr.io/getting-started/quickstarts/pubsub-quickstart/#step-3-subscribe-to-topics

**Expected content**
Change the app-port to 7006
`dapr run --app-id order-processor --resources-path ../../../components --app-port 7006 -- dotnet run`

**Comment by yaron2** [comment]
Thanks for reporting this, would be great to get a PR that fixes this.

**Comment by k3daevin** [comment]
Will do in a few hours

**Comment by k3daevin** [comment]
There is also a typo on https://docs.dapr.io/developing-applications/building-blocks/actors/actors-timers-reminders/#error-handling

> the runtime will contineu to invoke

Few hours turned into a few days

**Comment by kendallroden** [comment]
I think this is fixed

**Comment by kendallroden** [comment]
@hhunter-ms can you confirm? when i looked at the lastest it was updated

**Comment by hhunter-ms** [comment]
@kendallroden according to [the quickstart readme](https://github.com/dapr/quickstarts/blob/master/pub_sub/csharp/sdk/README.md), the .NET order-processor command needs updated in the docs, as well as:

- [JavaScript order-processor app-port](https://github.com/dapr/quickstarts/blob/master/pub_sub/javascript/sdk/order-processor/index.js#L6)
- [Go order-processor app-port](https://github.com/dapr/quickstarts/blob/master/pub_sub/go/sdk/order-processor/app.go#L23)

Also, this typo needs fixed - might as well lump into same PR:

> There is also a typo on https://docs.dapr.io/developing-applications/building-blocks/actors/actors-timers-reminders/#error-handling
> 
> > the runtime will contineu to invoke

🌟 🌟 UPDATE: .NET pub/sub app port was fixed [here](https://github.com/dapr/docs/pull/3753), so just JS and Go app ports need updated.

TL;DR:
- Update `dapr run` commands for the order-processor for JavaScript and Go here: https://docs.dapr.io/getting-started/quickstarts/pubsub-quickstart/#step-3-subscribe-to-topics
- Fix typo here: https://docs.dapr.io/developing-applications/building-blocks/actors/actors-timers-reminders/#error-handling
